Detections/AzureFirewall/SeveralDenyActionsRegistered.yaml

```diff
@@ -0,0 +1,34 @@
+id: 
+name: Several deny actions registered
+description: |
+  'This Creates an incident when a source has more than 1 registered deny action in Azure Firewall.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: AzureFirewall
+    dataTypes: 
+      - AzureDiagnostics
+queryFrequency: 1h
```

Comment:
Generally, we want defaults to be 1d on frequency and period.  You can leave it as 1h, but this may cause many results per day.

Reply:
I think it will take too long time to discover the attack pattern if this is only run with 1d frequency. 1h is the right frequency for this. It shouldn't create a lot of incidents as the firewall rules should be configured correct and this indicates an outgoing attack.